BE/PoroDev.Runtime/Extensions/ExceptionHelper.cs

```diff
@@ -0,0 +1,43 @@
+﻿using PoroDev.Common.Contracts;
+using PoroDev.Common.Exceptions;
+using PoroDev.Common.Models.RuntimeModels.Data;
+using PoroDev.Runtime.Extensions.Contracts;
+using static PoroDev.Runtime.Constants.Consts;
+
+namespace PoroDev.Runtime.Extensions
+{
+    public class ExceptionHelper
```

Comment:
Name of this helper class is kinda misleading. The purpose of its method is more than just exception handling/wrapping. Also should the class be static?